src/components/HomePage/Banner.js

```diff
@@ -137,8 +137,8 @@ function renderInfo(classes) {
 						component='h3'
 					>
 						<time dateTime={hothStart.toISOString()}>
-							{month} {startDay}{endDayString}, 2021
-						</time> | Virtual
+							{month} {startDay}{endDayString}, 2022
+						</time> | TBA
```

Comment:
```suggestion
						</time> | Location TBA
```

It's ambiguous as to what TBA refers to, so prepend the word "Location" for clarity